test-harness/src/main/scala/grasshopper/test/model/TestGeocodeModel.scala

```diff
@@ -15,11 +15,22 @@ object TestGeocodeModel {
   }
 
   case class PointInputAddressTract(pointInputAddress: PointInputAddress, geoid: String) {
+    override def toString = toCSV
     def toCSV: String = s"${pointInputAddress.inputAddress},${pointInputAddress.point.x},${pointInputAddress.point.y},${geoid}"
```

Comment:
Isn't this going to have problems when there's a comma in the `inputAddress`?  We should probably use real CSV lib.  In the meantime, you may want to at least wrap each element in double quotes.  Of course, you still have to worry about double quotes in the address string, but that's a lot less likely to happen.